package.json

```diff
@@ -51,6 +52,7 @@
     "marked-terminal": "1.4.0",
     "minimatch": "2.0.4",
     "mkdirp": "0.3.5",
+    "mocha": "2.2.5",
```

Comment:
mocha is both in dependencies and devDependencies with different versions. Which one is correct?